tests/components/omnilogic/test_config_flow.py

```diff
@@ -0,0 +1,15 @@
+"""Test the Omnilogic config flow."""
+from homeassistant import config_entries, setup
+from homeassistant.components.omnilogic.const import DOMAIN
+
+
+async def test_form(hass):
```

Comment:
You should have 100% coverage for config flows.

Reply:
Please address this.

Reply:
Working on this today. Limited knowledge of pytest, but I'm sure I can get it figured out.